docs/menu.yml

```diff
@@ -265,6 +265,10 @@ catalog:
                 path: "/en/setup/backend/backend-rabbitmq-monitoring"
               - name: "Kafka"
                 path: "/en/setup/backend/backend-kafka-monitoring"
+              - name: "Pulsar"
+                path: "/en/setup/backend/backend-pulsar-monitoring"
+              - name: "Bookkeeper"
+                path: "/en/setup/backend/backend-bookkeeper-monitoring"
```

Comment:
`Bookkeeper` -> `BookKeeper`. Case is sensitive for the name.

According to its website, 
> About
Apache BookKeeper - a scalable, fault tolerant and low latency storage service optimized for append-only workloads

This part should be in the database catalog.

Reply:
@liangyepianzhou This comment has not been resolved.

Reply:
Fixed